test/CMakeLists.txt

```diff
@@ -131,6 +131,7 @@ set(MULTI_TEST_SRC
     backend_sum.in.cpp
     backend_topk.in.cpp
     backend_arg_reduce.in.cpp
+    backend_graph_comparison.in.cpp
```

Comment:
This test requires that the CPU backend is built. Only add this source file if there is a CPU backend.

Reply:
Good point. Done.